<?xml version="1.0"?>
<launch>
	<!-- arguments -->
	<arg name="screen_x" default="1920"/>
	<arg name="screen_y" default="1080"/>

	<!-- load parameters -->
	<rosparam command="load" ns="/rcll/gamefield/" file="$(find rcll_status_board)/yaml/Gamefield.yaml" />

	<!-- Complete Status Board -->
	<node pkg="rcll_status_board" type="complete_status_board_node" name="complete" output="screen">
		<param name="screen_x" value="1920" />
		<param name="screen_y" value="1080" />
		<param name="fullscreen" value="false" />
		<param name="image_path" value="$(find rcll_status_board)/img/ppm/" />
		<param name="paging_time" value="15" />
		<param name="paging_wait_time" value="7" />
		<param name="shift_increase" value="5" />
		<param name="refbox_view" value="false" />
	</node>
</launch>
